src/Middleware/Diagnostics/src/ExceptionHandler/ExceptionHandlerFeature.cs

```diff
@@ -1,14 +1,16 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#nullable enable
+
 using System;
 
 namespace Microsoft.AspNetCore.Diagnostics
 {
     public class ExceptionHandlerFeature : IExceptionHandlerPathFeature
```

Comment:
💡 Consider modifying this to have a constructor instead of just read/write properties.

Reply:
It's going to be difficult to consider add a new public API to support nullability.